packages/retail-ui-extensions/src/components/Screen/Screen.ts

```diff
@@ -22,6 +23,7 @@ export interface ScreenProps {
   presentation?: ScreenPresentationProps;
   onNavigate?: () => void;
   onNavigateBack?: () => void;
+  onSecondaryActionPress?: () => void;
```

Comment:
Looking back at how these things are named, I with we had started with something like this instead:
- Naming `onSecondaryActionPress` as `onNavigateBack`.
- Naming `onNavigateBack` as `onNavigatedBack`. 
- Naming `onNavigate` as `onNavigatedHere`. 

As things stand, that would probably be a breaking change. For the present moment, what do we think about using a name like `overrideNavigateBack` instead? Although it will also be called not just for `back`, but a `close` action too, this follows the current naming of parameters and builds on it.